bson/bsoncodec/default_value_decoders_test.go

```diff
@@ -27,6 +27,10 @@ import (
 	"go.mongodb.org/mongo-driver/x/bsonx/bsoncore"
 )
 
+var (
+	defaultTestStructCodec = newDefaultStructCodec()
```

Comment:
IIUC, this is not necessary for these tests to pass, but just to avoid adding `structDescription`'s to the cache of the defaultStructCodec global, right?

Edit: reading more now, I now see that the global defaultStructCodec was removed...